Source/Scene/Cesium3DTileStyle.js

```diff
@@ -1657,32 +1657,32 @@ Cesium3DTileStyle.prototype.getColorShaderFunction = function (
   }
 
   this._colorShaderFunctionReady = true;
-  this._colorShaderFunction =
-    defined(this.color) && defined(this.color.getShaderFunction)
-      ? this.color.getShaderFunction(
-          functionHeader,
-          variableSubstitutionMap,
-          shaderState,
-          "vec4"
-        )
-      : undefined;
+  if (defined(this.color) && defined(this.color.getShaderFunction)) {
+    this._colorShaderFunction = this.color.getShaderFunction(
```

Comment:
@ptrgags @sanjeetsuhag Sorry for getting in here late, but this change has unexpected consequences because `this._colorShaderFunction` is not set back to `undefined` when `color` is set to `undefined`

See [this sandcastle](https://sandcastle.cesium.com/#c=dVLBbtswDP0VIZfaQCAj27BD4gbt2kuBARuSYLv4okpMI0QWA0pykAz791F2Wnhb6otMvke+5yd3ikRn4QgkboWHo3iAYFMrf/S9opnovn5AH5X1QM1kKn41XogIRNz5TthZAzR/HdQEKsJPJGc2A6Uop43/XS4a3/iO5aJ1ECD+rTccHx83A1j0Gonc294n9CsImEiD3BK294FpT6aYff70YTZW6I3LoMGDPJBtbbQdBKmMKS7KmXihnRHbDY6Bxl8KGeLJwfsu1xkefGp0yAlwWPmluCEwNyUnlbGww+NcRErwZnGtvNEqRAfZ1QbRPSv6kmJEz4GvoMUORK+e094mr6NlqLwEP7Yne0U2mbyBLV+QWYw5rdoPPh8txVOR5dnDZDqp++ll5ubnzrYHpJgDL6SsIrQHx7cYquek97xHh1AuBnJdjUdrYzthze2V/0Rop0JgZJucW9szf8yyrpj/36hDZax/+dYBOXXKtN1s+XVoSinrisvrk3HI7p/NfwA) which no longer works in master but works in 1.82